packages/network-core/src/Response.ts

```diff
@@ -154,7 +154,7 @@ export class Response<T extends TModel | Array<TModel>, TModel extends PureModel
       try {
         this.__data = initData(response, collection, overrideData);
       } catch (e) {
-        this.__internal.error = e;
+        this.__internal.error = e as any;
```

Comment:
Check if we can remove `as any` (I think there is another place with this identical line, probably in obsolete/network)